cmd/extract.go

```diff
@@ -25,13 +27,14 @@ import (
 )
 
 var targetDir string
+var buildah bool
 var extractContainerName string
 
 var extractCmd = &cobra.Command{
 	Use:   "extract",
 	Short: "Extract the stack and your Appsody project to a local directory",
 	Long: `This copies the full project, stack plus app, into a local directory
-in preparation to build the final docker image.`,
+in preparation to build the final buidlah / docker image.`,
```

Comment:
Typo of `buildah`. We should probably just generically refer to a container image though.

Reply:
done, thanks.